Library/PhotosController.swift

```diff
@@ -116,6 +143,19 @@ extension PhotosController: ViewerControllerDataSource {
     }
 }
 
+#if os(tvOS)
+    extension PhotosController: ViewerControllerDelegate {
+        func viewerController(_ viewerController: ViewerController, didChangeFocusTo indexPath: IndexPath) {}
+
+        func viewerControllerDidDismiss(_ viewerController: ViewerController) {
```

Comment:
When the controller is dismissed we tell the collection view to figure out what do focus on, this will trigger a call to `preferredFocusEnvironments`.